### PR TITLE
Update APIs to vRA 8.6.2

### PR DESCRIPTION
**Pull request by frodenas**
* Update swagger files to vRA v8.6.2
* Upgrade go-swagger to [v0.28.0](https://github.com/go-swagger/go-swagger/releases/tag/v0.28.0)
* Upgrade golang dependencies
* Add CMX Service API swagger
* Remove security parameters from Blueprint and IaaS swagger files

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -23,16 +24,16 @@ check-swagger:
...
 
-update: update-blueprint update-catalog-deployment update-iaas update-content update-pipeline update-project
+update: update-blueprint update-catalog-deployment update-iaas update-content update-pipeline update-project update-cmx
 
 update-blueprint:
 	curl --insecure 'https://${SWAGGER_ENDPOINT}/blueprint/api/swagger/swagger-api-docs?group=2019-09-12' | python3 -mjson.tool > swagger/vra-blueprint.json
 
 update-catalog-deployment:
-	curl --insecure 'https://${SWAGGER_ENDPOINT}/deployment/api/swagger/swagger/v2/api-docs?group=2019-01-15' | python3 -mjson.tool > swagger/vra-catalog-deployment.json
+	curl --insecure 'https://${SWAGGER_ENDPOINT}/deployment/api/swagger/swagger/v2/api-docs?group=2020-08-25' | python3 -mjson.tool > swagger/vra-catalog-deployment.json
```
Comment (dondon3345678) [inline Makefile:33]: Could you  share the documentation for checking this version?

File: hack/fix_blueprint_swagger [hunk 2]
```diff
@@ -33,15 +33,34 @@ def rename_blueprint_models(swagger):
         replace_value(swagger, "$ref", oldRef, newRef)
 
 
+def remove_global_security(swagger):
+    if 'securityDefinitions' in swagger:
+        del swagger['securityDefinitions']
+
+
+# See https://jira.eng.vmware.com/browse/TCS-4017 for context
```
Comment (dondon3345678) [inline hack/fix_blueprint_swagger:41]: Should we provide this link in the open source project?

File: hack/fix_cmx_swagger [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+
+# This is the location of the CMX swagger
+old_filename = "swagger/vra-cmx.json"
+new_filename = "swagger/vra-cmx-fixed.json"
+
+
+def replace_value(d, k, v, new):
+    if k in d and d[k] == v:
+        print(d[k])
```
Comment (dondon3345678) [inline hack/fix_cmx_swagger:14]: Remove debug messages

File: Makefile (hunk 1 above)
Comment (frodenas) [reply Makefile:33]: Go to any vRA 8.6.2 environment, and check the api-docs webpage, you will see the latest API version published. For reference, this is vRA Cloud: https://www.mgmt.cloud.vmware.com/deployment/api/swagger/swagger-ui.html?urls.primaryName=catalog

File: hack/fix_blueprint_swagger (hunk 2 above)
Comment (frodenas) [reply hack/fix_blueprint_swagger:41]: I don't see why not, there are already a few references to internal JIRA's, it's hard to keep track of them if we don't document them where there are needed. But if you think there's a better way to document it, just please let me know.

File: hack/fix_cmx_swagger (hunk 3 above)
Comment (frodenas) [reply hack/fix_cmx_swagger:14]: this is a copy & paster from the other scripts, I'll remove this and clean up the other scripts.

**Review by dondon3345678 — APPROVED** [review]
LGTM, approved.

**Pull request merged**